test/extensions/filters/network/rbac/integration_test.cc

```diff
@@ -0,0 +1,128 @@
+#include "extensions/filters/network/rbac/config.h"
+
+#include "test/integration/integration.h"
+#include "test/test_common/environment.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace RBAC {
+
+std::string rbac_config;
+
+class RoleBasedAccessControlNetworkFilterIntegrationTest
+    : public BaseIntegrationTest,
+      public testing::TestWithParam<Network::Address::IpVersion> {
+public:
+  RoleBasedAccessControlNetworkFilterIntegrationTest()
+      : BaseIntegrationTest(GetParam(), rbac_config) {}
+
+  static void SetUpTestCase() {
+    rbac_config = R"EOF(
+admin:
+  access_log_path: /dev/null
+  address:
+    socket_address:
+      address: 127.0.0.1
+      port_value: 0
+static_resources:
+  clusters:
+    name: cluster_0
+    hosts:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 0
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 8000
```

Comment:
do not use fixed port number for tests (they couldn't be ran parallel, and fail for port conflict, which is hard to notice)

Reply:
Updated, we don't really need port in rbac config.